### PR TITLE
Feature/workflow description

### PR DESCRIPTION
**Pull request by aleksey-pastuhov**
* Extend documentation structure
* Git flow description
* Feature flow description
* Project Management description

### CODE REVIEW

**Comment by aleksey-pastuhov** [comment]
[Workflow description](https://trello.com/c/SZzFI4ch/15-workflow-description)

File: SUMMARY.md [hunk 1]
```diff
@@ -1,6 +1,37 @@
 # Summary
 
 * [Introduction](README.md)
-* [General Description](general-description/intro.md)
-* [Architecture](diagrams/architecture.md)
+* Project Documentation
+  * [Project Overview](project-documentation/project-overview.md)
+  * Product Requirements
+  * Diagrams
+  * Features overview
```
Comment (vpantasenko) [inline SUMMARY.md]: Features Overview (capitalise)

File: development/common-processes/git-flow.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
+# Git flow
+
+We use simple [git-flow](http://nvie.com/posts/a-successful-git-branching-model/). You can find down a short description of the flow.
+
+## Branches
+
+Instead of a single master branch, this workflow uses two branches to record the history of the project. The master branch stores the official release history, and the develop branch serves as an integration branch for features.
```
Comment (vpantasenko) [inline development/common-processes/git-flow.md]: ...two branches to track project history: `master` and `development`.

File: development/common-processes/git-flow.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+## Feature implementation
+
+For every task developer makes, he must to:
```
Comment (vpantasenko) [inline development/common-processes/git-flow.md]: While working on a task, make sure that you follow these rules:
* New feature (bugfix, hotfix, etc) branch is created from a `develop` branch;
* Branch name is the same as feature name and has `feature/` prefix. For example, branch for "Add navigation sidebar" feature will be named `feature/add-navigation-sidebar`;
* One feature can have only one branch related to it;
* Assign stakeholders to the Pull request.

File: development/common-processes/git-flow.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+feature/modal
+```
+
+As soon as your job is done - you can make a pull request.
+
+How to ensure that your code is ready for review:
+
+* It is stable (no failures, no simple/stupid bugs);
+* It meets requirements of your task;
+* All linters(PEP, FLAKE, ESLINT, SASSLINT, etc.) and tests passed.
```
Comment (vpantasenko) [inline development/common-processes/git-flow.md]: Please, add space between "linters" word and parenthesis.

File: sdlc/processes/feature-flow.md [hunk 5]
```diff
@@ -0,0 +1,19 @@
+# Feature flow
+
+Stages of working with features:
+
+1. Definition
```
Comment (vpantasenko) [inline sdlc/processes/feature-flow.md]: Please, make all list items enumerable.

File: SUMMARY.md [hunk 6]
```diff
@@ -1,6 +1,37 @@
...
+  * Versions
 
+* SDLC
+  * Environments
+  * Processes
+    * [Feature flow](sdlc/processes/feature-flow.md)
+  * Release Management
+
+* Development
+  * Architecture overview
```
Comment (pawlyk) [inline SUMMARY.md]: Why you have here two `Architecture overview`?

File: development/common-processes/git-flow.md [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+
+## Branches
+
+Instead of a single master branch, this workflow uses two branches to record the history of the project. The master branch stores the official release history, and the develop branch serves as an integration branch for features.
+
+## Feature implementation
+
+For every task developer makes, he must to:
+
+* Create feature branch from a `develop` branch
```
Comment (pawlyk) [inline development/common-processes/git-flow.md]: We need additionally discuss this flow.

File: project-management/team-register.md [hunk 8]
```diff
@@ -0,0 +1,8 @@
+# Team register
+
+Capacity: ???
+
+| **Member**       | - | Paul Stadnikov | Taras Rudnyk | Valerii Tryhubov | Aleksey Pastuhov | Volodymyr Pantasenko |
```
Comment (pawlyk) [inline project-management/team-register.md]: I think we do not need this. Anyway we need discuss this.

File: sdlc/processes/feature-flow.md (hunk 5 above)
Comment (pawlyk) [inline sdlc/processes/feature-flow.md]: I think we should make this process more flexible. For some cases, we do not need all team in research-discuss-planned-implementation phase for some feature.

**Comment by pawlyk** [comment]
Please rename project repository name to `development-docs` or something like this, because in the future we can use doc name for our common documentation.

File: SUMMARY.md (hunk 1 above)
Comment (aleksey-pastuhov) [inline SUMMARY.md]: Fixed

File: development/common-processes/git-flow.md (hunk 2 above)
Comment (aleksey-pastuhov) [inline development/common-processes/git-flow.md]: Fixed

File: development/common-processes/git-flow.md (hunk 3 above)
Comment (aleksey-pastuhov) [inline development/common-processes/git-flow.md]: Fixed!

File: development/common-processes/git-flow.md (hunk 4 above)
Comment (aleksey-pastuhov) [inline development/common-processes/git-flow.md]: Fixed!

File: sdlc/processes/feature-flow.md (hunk 5 above)
Comment (aleksey-pastuhov) [inline sdlc/processes/feature-flow.md]: @vpantasenko No need to do it. In markdown if there is at least first item has number - all other list items will have numbers, even if they have asterisks.

File: SUMMARY.md (hunk 6 above)
Comment (aleksey-pastuhov) [inline SUMMARY.md]: Sorry) Copy-paste fixed! :)

File: development/common-processes/git-flow.md (hunk 7 above)
Comment (aleksey-pastuhov) [inline development/common-processes/git-flow.md]: Okay, I thought that git-flow is a standard.

File: project-management/team-register.md (hunk 8 above)
Comment (aleksey-pastuhov) [inline project-management/team-register.md]: okay

**Comment by aleksey-pastuhov** [comment]
@pawlyk I will rename after merge of PR

File: sdlc/processes/feature-flow.md (hunk 5 above)
Comment (vpantasenko) [inline sdlc/processes/feature-flow.md]: @aleksey-pastuhov Got it, thanks!

Comment (aleksey-pastuhov) [inline sdlc/processes/feature-flow.md]: @pawlyk  I've added next text:

All stages except discussion with a team can be made(or even skipped) by a single developer.

**Pull request merged**